src/network/discovery.js

```diff
@@ -135,7 +135,9 @@ createNameSpace("realityEditor.network.discovery");
 
     exports.setPrimaryWorld = (ip, id) => {
         primaryWorld = {
-            ip: ip,
```

Comment:
Is there nothing that uses the ip value here?

Reply:
The code does use the ip value if present (in src/network/index.js). However, I think that setting the ip address is what causes joining an existing scan from the metaverse manager to fail.

More precisely, the following:
device A and device B are logged into realitydemo@ptc.com in the metaverse manager
device A creates a new scan
device B sees the scan in the metaverse manager, clicks Join Metaverse
right now: device B doesn't localize or download the area target
with this change (expected, can't test locally right now): device B successfully sees that device A is hosting the expected world id

Reply:
I decided to drop this blunt approach in favor of selectively not providing `ip` in callers of this function